reana_server/config.py

```diff
@@ -428,6 +428,11 @@ def _get_rate_limit(env_variable: str, default: str) -> str:
     )
 """Maximum allowed period (in days) for interactive session inactivity before automatic closure."""
 
+REANA_INTERACTIVE_SESSIONS_ENVIRONMENTS = json.loads(
```

Comment:
Can we also expose to the users in the `info` output whether the server allows to specify custom Jupyter images by users? See the `interactive_sessions.environments.jupyter.allow_custom` Helm value.